index.html

```diff
@@ -0,0 +1,173 @@
+    <header class="flex justify-end p-spacing6">
+      <nav class="flex">
+        <ul class="flex items-center space-x-15">
+          <li>
+            <a
+              href="https://mail.google.com/mail/&ogbl"
+              class="text-gray-900 hover:underline text-opacity-87"
+              >Gmail</a
+            >
+          </li>
+          <li>
+            <a
+              href="/https://www.google.rw/imghp?hl=en&ogbl"
+              class="text-gray-900 hover:underline text-opacity-87"
+              >Images</a
+            >
+          </li>
+          <li class="flex items-center">
+            <div class="flex p-1">
+              <a
+                href="/#"
+                class="flex-1 w-10 h-10 p-2 hover:rounded-full hover:bg-gray-300"
+                ><img
+                  src="./svgs/menu-icon.svg"
+                  alt="menu-icon"
+                  class="w-6 h-6"
+              /></a>
+            </div>
+            <a href="/#" class="p-1 rounded-full hover:bg-gray-300">
+              <img
+                alt="user icon"
+                class="w-6 h-6 p-4 bg-green-600 rounded-full"
+                src="https://lh3.googleusercontent.com/ogw/AOh-ky2yQRUjsCUAvLNrKYDRoO859IsAndSF4a80H7iu=s32-c-mo"
+              />
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="flex flex-col items-center flex-1 px-2 sm:px-0">
+      <img
+        src="./images/googlelogo_color_272x92dp.png"
+        alt="google icon"
+        class="object-cover object-center mt-16 lg:mt-10"
+      />
+      <div class="p-7 sm:w-11/12 md:w-1/2 md:min-w-lg">
+        <form class="flex flex-col items-center space-y-7 pt-s pt-spacing6">
+          <div
+            class="flex items-center w-full px-4 py-2 space-x-6 bg-white border border-gray-300 rounded-4xl hover:shadow-search hover:border-gray-400"
+          >
+            <img
+              src="./svgs/search-icon.svg"
+              alt="search-icon"
+              class="w-6 h-6"
+            />
+
+            <label for="search-text" class="sr-only"></label>
+            <input
+              type="text"
+              id="search-text"
+              class="flex-1 block w-full px-3 py-1.5 text-sm font-normal text-black text-opacity-87 bg-transparent bg-clip-padding border-solid border-0 transition ease-in-out m-0 focus:bg-transparent focus:border-0 focus:outline-none"
+            />
+            <img
+              src="./svgs/micro-icon.svg"
+              alt="micro icon"
+              class="w-6 h-6 hover:cursor-pointer"
+            /><img
+              src="./svgs/google-camera.svg"
+              alt="google camera icon"
+              class="w-6 h-6 hover:cursor-pointer"
+            />
+          </div>
+          <div class="flex items-center space-x-5">
+            <button
+              class="px-2 py-2 text-sm text-gray-900 border border-gray-200 rounded bg-gray-50 sm:px-4 hover:border-gray-300"
```

Comment:
the original google gets border only on hover correct and also on the next button